sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -971,8 +971,8 @@ def test_neighbors_badargs():
         assert_raises(ValueError,
                       nbrs.predict,
                       [[]])
-        if (isinstance(cls, neighbors.KNeighborsClassifier) or
-                isinstance(cls, neighbors.KNeighborsRegressor)):
+        if (isinstance(cls(), neighbors.KNeighborsClassifier) or
+                isinstance(cls(), neighbors.KNeighborsRegressor)):
```

Comment:
Maybe use `issubclass` to avoid instantiating it?

Reply:
Done